src/main/java/gov/cabinetoffice/gap/adminbackend/controllers/ApplicationFormSectionsController.java

```diff
@@ -166,6 +167,31 @@ public ResponseEntity updateSectionStatus(final HttpServletRequest request,
         }
     }
 
+    @PatchMapping("/order")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "Section order updated successfully.",
+                    content = @Content(mediaType = "application/json")),
+            @ApiResponse(responseCode = "403", description = "Insufficient permissions to update section.",
+                    content = @Content(mediaType = "application/json")),
+            @ApiResponse(responseCode = "404", description = "No application or section found with given id.",
+                    content = @Content(mediaType = "application/json")) })
+    public ResponseEntity<String> updateSectionOrder(final HttpServletRequest request,
+            final @PathVariable Integer applicationId,
+            final @RequestBody ApplicationSectionOrderPatchDto sectionOrderPatchDto) {
+        try {
+            this.applicationFormSectionService.updateSectionOrder(applicationId, sectionOrderPatchDto.getSectionId(),
+                    sectionOrderPatchDto.getUpOrDown());
+            logApplicationUpdatedEvent(request.getRequestedSessionId(), applicationId);
```

Comment:
Apparently sonarlint prefers due to a security issue:
```suggestion
            logApplicationUpdatedEvent(request.getSession().getId(), applicationId);
```

Reply:
Fixed in latest commit